cafebabel/templates/articles/translations/create.html

```diff
@@ -5,16 +5,19 @@
   {{ extra_js(['js/articles-tags.js', 'js/textarea-change.js']) }}
 {% endblock %}
 
+{% block body_class %}article-page-edit{% endblock %}
 {% block body %}
-<h1>Creating a translation in {{ language }} for article “{{ article.title }}”</h1>
 
 <form method=post action=.>
   <input type=hidden name=language value="{{ language }}">
   <input type=hidden name=original value="{{ article.id }}">
-  <div class=text>
+  {% if article.has_image %}
+    <img src="{{ article.image_url }}" alt="Image for {{ article.title }}">
+  {% endif %}
+  <h1 class="edit active">
     <label for=title>Title</label>
-    <input id=title name=title required>
-  </div>
+    <input id=title name=title required placeholder="Translation in {{ language }} for article “{{ article.title }}”">
```

Comment:
Using it with real data, the whole line is hardly visible given the size of the text. Maybe we could put that indication somewhere else where it's kept during edition (above or below the field)?

Reply:
#244 ? not a priority I assume